src/addrman.h

```diff
@@ -21,6 +21,7 @@
 #include <set>
 #include <stdint.h>
 #include <streams.h>
+#include <unordered_map>
```

Comment:
nit: looks like you can move the `<map>` include to `addrman.cpp` now.

Reply:
Right! And even better - `std::map` is only used inside `CAddrMan::Check_()` in a way that fits `unordered_map` better - we only insert elements, query elements by key and check container size.

So I changed `CAddrMan::Check_()` to use `unordered_(set|map)` and removed the `#include <map>`.